source/developer/style-guide.md

```diff
@@ -127,18 +125,18 @@ function wrongGetStr(stuff) {
 
 ## React-JSX
 
-Part of the build process is running ESLint. ESLint is the final authority on all style issues. PRs will not be accepted unless there are no errors running ESLint. The ESLint configuration file can be found in: [webapp/.eslintrc](https://github.com/mattermost/mattermost-webapp/blob/master/.eslintrc.json)
+Part of the build process is running ESLint. ESLint is the final authority on all style issues. PRs will not be accepted unless there are no errors running ESLint. The ESLint configuration file can be found in: [webapp/.eslintrc](https://github.com/mattermost/mattermost-webapp/blob/master/.eslintrc.json).
 
-Instructions on how to use ESLint with your favourite editor can be found here: [http://eslint.org/docs/user-guide/integrations](http://eslint.org/docs/user-guide/integrations)
+Instructions on how to use ESLint with your favourite editor can be found here: [http://eslint.org/docs/user-guide/integrations](http://eslint.org/docs/user-guide/integrations).
 
-You can run eslint using the makefile by using `make check-style`
+You can run ESlint using the makefile by using `make check-style`.
 
 The following is a subset of what ESLint checks for. ESLint is always the authority. 
 
 ### General
 
 - Include only one React component per file.
-- Use class \<name\> extends React.Component over React.createClass unless you need mixins
+- Use class \<name\> extends React.Component over React.createClass unless you need mixins.
```

Comment:
```suggestion
- Use class \<name\> extends `React.Component` over `React.createClass` unless you need mixins.
```

I'm unsure whether I've formatted this correctly or whether it should be "Use `class \<name\> extends React.Component`.

Reply:
I'm not entirely sure, but your's looks correct. I wonder if the word `extends` is meant to be there

Reply:
I was also wondering that but am not familiar enough with the language to make the call. I've looked up the areas of expertise and perhaps @saturninoabril can help? :)

Reply:
Yup, it should be there, actually starting with class.
```suggestion
- Use `class <Name> extends React.Component` over `React.createClass` unless you need mixins.
```